### PR TITLE
Refresh the docs with better language

### PR DESCRIPTION
**Pull request by yurishkuro**
## Description of the changes
- Remove some references to OpenTracing
- Provide links to HotROD blog
- Update wording on the home page and some intro pages

### CODE REVIEW

File: data/indexpage.yaml [hunk 1]
```diff
@@ -1,13 +1,11 @@
...
+- text: track down root causes
   icon: call_split
-- text: service dependency analysis
+- text: analyze service dependencies
   icon: format_align_center
-- text: distributed context propagation
-  icon: wifi
 why: |
-  As on-the-ground microservice practitioners are quickly realizing, the majority of operational problems that arise when moving to a distributed architecture are ultimately grounded in two areas: **networking** and **observability**. It is simply an orders of magnitude larger problem to network and debug a set of intertwined distributed services versus a single monolithic application.
+  Distributed tracing observability platforms, such as Jaeger, are essential for modern software applications that are architected as microservices. Jaeger maps the flow of requests and data as they traverse a distributed system. These requests may make calls to multiple services, which may introduce their own delays or errors. Jaeger connects the dots between these disparate components, helping to identify performance bottlenecks, troubleshoot errors, and improve overall application reliability. Jaeger is 100% open source, cloud native, and infinitely scalable.
```
Comment (yurishkuro) [inline data/indexpage.yaml:11]: This was fun to write with the help of Bard, and using the writing styles of Alan Turing and Leslie Lamport.

File: content/docs/next-release/getting-started.md [hunk 2]
```diff
@@ -65,54 +65,46 @@ Port  | Protocol | Component | Function
...
-#### From Source
+#### With Docker Compose
 
 ```bash
-mkdir -p $GOPATH/src/github.com/jaegertracing
-cd $GOPATH/src/github.com/jaegertracing
 git clone git@github.com:jaegertracing/jaeger.git jaeger
-cd jaeger
-go run ./examples/hotrod/main.go all
+cd jaeger/examples/hotrod
```
Comment (albertteoh) [inline content/docs/next-release/getting-started.md:102]: This also requires a git clone just like `From Source`:
```
git clone git@github.com:jaegertracing/jaeger.git jaeger
```

**Review by albertteoh — COMMENTED** [review]
Looks good, just added a couple of comments.

File: content/docs/next-release/getting-started.md [hunk 3]
```diff
@@ -65,54 +65,46 @@ Port  | Protocol | Component | Function
...
--   Use open source libraries with OpenTracing integration to get
-    vendor-neutral instrumentation for free.
-
-### Prerequisites
+### Running
 
--   You need [Go toolchain](https://golang.org/doc/install) installed on your machine to run from source
-    (see [go.mod](https://github.com/jaegertracing/jaeger/blob/master/go.mod) file for required Go version).
--   Requires a [running Jaeger backend](#all-in-one) to view the traces.
+We recommend running Jaeger and HotROD via `docker compose`. In order to run it from source you need:
```
[the remark this answers is not in the document]
Comment (yurishkuro) [reply content/docs/next-release/getting-started.md:92]: good call

[the remark this answers is not in the document]
Comment (yurishkuro) [reply content/docs/next-release/getting-started.md:92]: PTAL https://github.com/jaegertracing/documentation/pull/650/files

**Pull request merged**